README.md

```diff
@@ -19,6 +21,13 @@ doxygen Doxyfile
 firefox doc/html/index.html
 ```
 
+###CMake Supported###
+
+Clone the project to some local path and add the following line to your `CMakeLists.txt`:
+
+    find_package(elegant-progressbars "<ABSOLUTE-PATH-TO-ELEGANT-PROGRESSBAR>/cmake")
```

Comment:
Does giving a path even work without specifying PATHS?
I always thought you had to choose between the 2 forms:
```cmake
find_package(elegant-progressbars PATHS "<ABSOLUTE-PATH-TO-ELEGANT-PROGRESSBAR>/cmake")
find_package(elegant-progressbars)
```